addons/base_import/static/src/js/import.js

```diff
@@ -220,6 +220,11 @@ openerp.base_import = function (instance) {
             this.$('.oe_import_button, .oe_import_file_reload')
                     .prop('disabled', true);
             this.$el.addClass('oe_import_with_file');
+            if (!(this.$('input.oe_import_file').val().split('.').pop() == "csv")) {
+                this.$el.find('.oe_import_toggle').hide();
+            }else {
+                this.$el.find('.oe_import_toggle').show();
+            }
```

Comment:
If file is kind of xls or xlsx or ods then there is no need of Separator and Quoting, but I think Encoding is still required